code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -572,6 +572,7 @@
 		revvie.key = O.key
 		revvie.mind.assigned_role = "revenant"
 		revvie.mind.special_role = "Revenant"
+		revvie.mind.gate_restriction = BOTH_GATES

```

Comment:
STATION_GATE
If the revenant is somehow on an away mission, it should be possible for it to escape.